packages/runtime/container-runtime/src/pendingStateManager.ts

```diff
@@ -356,10 +356,7 @@ export class PendingStateManager implements IDisposable {
 							message,
 							{
 								runtimeVersion: pkgVersion,
-								batchClientId:
-									this.pendingBatchBeginMessage.clientId === null
-										? "null"
-										: this.pendingBatchBeginMessage.clientId,
+								batchClientId: this.pendingBatchBeginMessage.clientId ?? "null",
```

Comment:
This would be a change of behavior for `this.pendingBatchBeginMessage.clientId === undefined`. Not sure if it's possible but I'd probably revert this one. Same thing for the two changes in `scheduleManager.ts` next to this file.